x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/fields/field_types/boolean_type.tsx

```diff
@@ -70,7 +70,8 @@ export const BooleanType = ({ field }: Props) => {
           description={i18n.translate(
             'xpack.idxMgmt.mappingsEditor.booleanNullValueFieldDescription',
             {
-              defaultMessage: 'Whether to substitute values for any explicit null values.',
+              defaultMessage:
+                'Replace explicit null values with the selected boolean value so that it can be indexed and searched.',
```

Comment:
I think "the selected boolean value" lacks context when the toggle is disabled, since the select element is hidden. How about changing it to "a specific boolean value"?